scalability_and_performance/scaling-worker-latency-profiles.adoc

```diff
@@ -16,8 +16,6 @@ toc::[]
 
 include::snippets/worker-latency-profile-intro.adoc[]
 
-You can configure worker latency profiles when installing a cluster or at any time you notice increased latency in your cluster network.
```

Comment:
This might be nice to still keep

Reply:
Think a section on enabling this after cluster creation would be good

Reply:
@paigerube14  I agree - however, we don't know that it's possible to configure after installation, right?

Reply:
By default on a cluster there is no profile defined 

```
% oc get nodes.config/cluster
apiVersion: config.openshift.io/v1
kind: Node
metadata:
 name: cluster
spec: {}
```

I added the `workerLatencyProfile: Default` under spec and saw values change from the 
Before 
```
%  oc get KubeControllerManager -o yaml | grep -A 1 node-monitor
% oc get KubeAPIServer -o yaml | grep -A 1 default-
```
After updating to default profile 
 ```
% oc get KubeAPIServer -o yaml | grep -A 1 default-             
        default-not-ready-toleration-seconds:
        - "300"
        default-unreachable-toleration-seconds:
        - "300"
 %  oc get KubeControllerManager -o yaml | grep -A 1 node-monitor
        node-monitor-grace-period:
        - 40s
```

Saw values change for both medium and low profiles as well 

I am not sure if that is supported since I didnt see the kubelet information change (think this is a bug but will ask the test case owner) 
 ```
% oc debug node/ip-10-0-145-121.us-east-2.compute.internal 
Temporary namespace openshift-debug-nt46w is created for debugging node...
Starting pod/ip-10-0-145-121us-east-2computeinternal-debug ...
To use host binaries, run `chroot /host`
chroot /host
Pod IP: 10.0.145.121
If you don't see a command prompt, try pressing enter.
sh-4.4# chroot /host
sh-5.1# cat /etc/kubernetes/kubelet.conf | grep nodeStatusUpdateFrequency 
  "nodeStatusUpdateFrequency": "0s",
```

Reply:
@paigerube14 This is completed - I've added "How to see the values" at the end of the "Using..." document.